docs/monitors/kubernetes-cluster.md

```diff
@@ -77,6 +77,10 @@ Metrics that are categorized as
 (*default*) are ***in bold and italics*** in the list below.
 
 
+ - `kubernetes.container_cpu.limit` (*gauge*)<br>    Maximum CPU limit set for the container
```

Comment:
Change the `.` to `_` so that the metric names are e.g. `kubernetes.container_cpu_limit`.

Reply:
Updated metric names